tst/Boolean/Assert-False.Tests.ps1

```diff
@@ -1,13 +1,13 @@
 Describe "Assert-False" {
-        It "Passes when given `$false" {
+        It "Passes when cmdlet Assert-False is given `$false" {
             $false | Assert-False
         }
 
-        It "Passes when given falsy" -TestCases @(
```

Comment:
I like expanding the value in the description, but doing it from the actual value is just fine (use <actual> btw not <Actual> if you will fix it). Pester won't handle it correctly for the moment but I am okay with that. There are few other tests that would need to be changed as well, but I will rather focus on getting the functionality that shows the data correctly in pester, than manually writing the values out in extra key in hashtable. Again stating the name in the test name is not necessary.